main.py

```diff
@@ -1,10 +1,69 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
+def create_race(race_data: dict) -> Race | None:
+    race_description = race_data.get("description")
+    if race_description is None:
+        return None
```

Comment:
You should not abandon race creation in case a description is absent. Create or retrieve objects as is based on the `name` and `description` from the data.